build/publish-mrt.yml

```diff
@@ -136,11 +140,19 @@ steps:
       mrm.h
     TargetFolder: '$(Build.ArtifactStagingDirectory)\fullnuget\include'
 
+- script: |
+    dir /s $(Build.ArtifactStagingDirectory)\fullnuget
+  displayName: HK 2
+
 - task: PublishBuildArtifacts@1
   inputs:
     PathtoPublish: '$(Build.ArtifactStagingDirectory)\fullnuget'
     ArtifactName: 'FullNuget'
 
+- script: |
+    dir /s $(Build.ArtifactStagingDirectory)\fullnuget
+  displayName: HK 3
```

Comment:
Here

Reply:
Yup 3rd of the trinity. See other comment. Thanks